app/controllers/orders_controller.rb

```diff
@@ -32,6 +32,17 @@ def index
     @user = current_user
   end
 
+  def update
+    order = Order.find(params[:id])
+    order.update(status: "Cancelled")
+    # item_array = Item.find(order.item_orders.where(fulfilled: true).pluck(:item_id))
+    # order.item_orders.joins(:item).where(fulfilled: true)
+    binding.pry
```

Comment:
Can lines 38-40 be deleted?

Reply:
Actually we're still working on this so I think we should keep it. The comment are things we tried but couldn't get to work, it's part of story 27